app/build.gradle

```diff
@@ -32,10 +32,12 @@ android {
 }
 
 dependencies {
-
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.2.1'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+
+    testImplementation 'junit:junit:4.13.1'
+    testImplementation 'org.mockito:mockito-core:3.3.3'
     testImplementation 'junit:junit:4.+'
```

Comment:
check that you have this dependency repeted

Reply:
Solved!